packages/clay-pagination/src/Pagination.tsx

```diff
@@ -18,10 +18,13 @@ interface IProps extends React.HTMLAttributes<HTMLUListElement> {
 
 const ClayPagination = React.forwardRef<HTMLUListElement, IProps>(
 	({children, className, size, ...otherProps}: IProps, ref) => {
+		const ariaLabel = otherProps['aria-label'] || 'Pagination';
+
 		return (
-			<nav aria-label={otherProps['aria-label'] || 'Pagination'}>
+			<nav aria-label={ariaLabel}>
 				<ul
 					{...otherProps}
+					aria-label={undefined}
```

Comment:
@pat270 considering @marcoscv-work's comment I think the best thing here would be to move the `aria-label` to the `ul`, I did a quick test with VoiceOver and really when focusing on the links the `aria-label` in the `nav ` will not be announced.

Reply:
@matuzalemsteles I didn't understand the comment, but I do now. I think we should leave it on both, as @marcoscv-work said. When I use `ctrl + option + arrow` to navigate, it just announces navigation when it focus the `nav`. It's more descriptive if Pagination is there.